src/ProjectNeon/Assets/Features/DeckBuilder/Scripts/DeckController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class DeckController : MonoBehaviour
+{
+    [SerializeField] DeckBuilderState deckBuilderState;
+    [SerializeField] List<DeckVisualizer> deckVisualizer;
+
+    private void Start()
```

Comment:
All Event Subscriptions must happen in `OnEnable`.


You are missing the corresponding `OnDisable` unsubscribe.

That leads to this error which happens when you click "Save" and then back to "Customize Deck"

![image](https://user-images.githubusercontent.com/10834339/66425417-78bc9f00-e9c4-11e9-9fff-ab4c60a2f6ce.png)